xblock/test/test_mixins.py

```diff
@@ -473,3 +478,58 @@ def test_datetime_serialization(self, value, expected_attributes):
         self.test_xblock_datetime.add_xml_to_node(node)
 
         self._assert_node_attributes(node, expected_attributes)
+
+
+class CompletableXBlockMixinTest(TestCase):
+    class TestBuddyXBlock(XBlock, CompletableXBlockMixin):
+        pass
+
+    def _make_block(self, runtime=None):
+        """ Creates a test block """
+        runtime = runtime if runtime else mock.Mock(spec=Runtime)
+        scope_ids = ScopeIds("user_id", "test_buddy", "def_id", "usage_id")
+        return self.TestBuddyXBlock(runtime=runtime, field_data=DictFieldData({}), scope_ids=scope_ids)
+
+    def test_has_custom_completion_property(self):
+        block = self._make_block()
+        self.assertTrue(block.has_custom_completion)
+        self.assertTrue(getattr(block, 'has_custom_completion', False))
```

Comment:
What's the benefit of getting the block both through direct attribute access and through getattr?